server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -896,11 +896,15 @@ public DiscoveryStats stats() {
         ClusterStateStats clusterStateStats = clusterManagerService.getClusterStateStats();
         ArrayList<PersistedStateStats> stats = new ArrayList<>();
         Stream.of(PersistedStateRegistry.PersistedStateType.values()).forEach(stateType -> {
-            if (persistedStateRegistry.getPersistedState(stateType) != null
-                && persistedStateRegistry.getPersistedState(stateType).getStats() != null) {
-                stats.add(persistedStateRegistry.getPersistedState(stateType).getStats());
+            if (persistedStateRegistry.getPersistedState(stateType) != null) {
+                if (persistedStateRegistry.getPersistedState(stateType).getUploadStats() != null) {
```

Comment:
Nit: && operator can be used to join these conditions.

Reply:
done